crt_portal/cts_forms/templates/forms/complaint_view/resources/resource_table.html

```diff
@@ -94,10 +99,9 @@
                         {% with last=datum.resource.contact_last_name|default:"—" %}
                         {{ last|truncatechars:20 }}{% endwith %}, {% with first=datum.resource.contact_first_name|default:"—" %}
                         {{ first|truncatechars:20 }}
-                        {% endwith %}
-                        {{ datum.resource.contact_title }}
-                        {{ datum.resource.contact_affiliation }}
-                        {{ datum.resource.contact_email }}
+                        {% endwith %} <br/>
```

Comment:
nit: consider using `<p>` instead of `<br/>` for better semantics

https://stackoverflow.com/a/13688203/13231999